src/assets/translations/it/labels.js

```diff
@@ -2,40 +2,64 @@ export const labels = {
   address: "Indirizzo",
   amount: "Importo",
   availables: "Disponibili",
+  bonded: "Vincolati",
   commission: "Commissione",
   commissionRate: "Tasso commissione",
+  cumulativeShare: "% Quota cumulata",
+  date: "Data",
   ddoReference: "Riferimento DDO",
   delegated: "Delegati",
+  delegator: "Delegante",
   delegatorAddress: "Delegante",
   depositEndTime: "Termine deposito",
   description: "Descrizione",
+  details: "Dettagli",
   did: "DID",
+  fee: "Onorario",
+  finalDate: "Data finale",
   firstUser: "Primo utente",
   fromAddress: "Da indirizzo",
+  gasUsedWanted: "Gas (Utilizzato/Richiesto)",
+  hash: "Hash",
+  height: "Altezza",
   identity: "Identità",
   keyType: "Tipo chiave",
   keyValue: "Valore chiave",
   metadata: "Metadati",
   minSelfDelegation: "Min. auto delega",
   operatorAddress: "Operatore",
+  others: "Altri",
   outstandingRewards: "Premi straordinari",
   owner: "Proprietario",
+  price: "Prezzo",
   proposer: "Proponente",
   publicKey: "Chiave pubblica",
+  rank: "Rango",
   receiver: "Destinatario",
   reference: "Riferimento",
+  result: "Risultato",
   rewards: "Ricompense",
   secondUser: "Secondo utente",
+  self: "Self",
   sender: "Mittente",
+  share: "Quota",
   signer: "Firmatario",
+  status: "Status",
   submitTime: "Orario di presentazione",
+  time: "tempo",
```

Comment:
Se questa e' la traduzione usata per il tempo dall'ultimo blocco, possiamo cambiarla in "Tempo dall'ultimo blocco"? Non credo sia molto chiara al momento